smartsim/_core/_install/builder.py

```diff
@@ -462,7 +447,7 @@ def build_from_git(self, git_url: str, branch: str, device: str = "cpu") -> None
                 "634916c722e718cc6ea3fad46e63f7d798f9adc2",
             ]
         else:
-            # Clone RAI release commit
+            # Clone RAI release commit for versions > 1.2.7
```

Comment:
nitpick: I think the comment doesn't accurately reflect that a version both LT and GT 1.2.7 will be cloned here